packages/advjs/src/client/components/adv/AdvBlack.vue

```diff
@@ -54,9 +54,7 @@ watch(() => props.content, () => {
 <template>
   <div
     flex="~ col"
-    class="adv-black absolute justify-center items-center"
```

Comment:
This is the atrributify feature, the class is too long and not easy to read.

Reply:
OK, I see.